07week/addressBook.js

```diff
@@ -0,0 +1,32 @@
+let arrayOfContacts = [];
+
+window.onload = function() {
+  getContacts()
+}
+
+const getContacts = () => {
+  fetch('https://randomuser.me/api/?results=10')
+        .then(function (res) {return res.json()})
+        .then(contacts => arrayOfContacts = contacts)
+}
+
+const consoleContacts = () => {
+  console.log(arrayOfContacts)
+}
+
+const displayContacts = function() {
+  const allContacts = document.getElementById('all-contacts');
+  arrayOfContacts.results.map(function(element) {
+    const li = document.createElement('li')
+    const text = document.createTextNode(`Title: ${element.name.title}.  ${element.name.first} ${element.name.last} 
```

Comment:
I would suggest using different element for the different attributes instead of a single element that has all the details